### ISSUE TITLE
UDP Server and Client Samples

### ISSUE THREAD

**Issue by B03901108** [MCUSec/SEmu#7]
Dear SEmu developers,
I cloned the SEmu GitHub repo and checked SEmu/DataSet/fuzz_tests/f429.
The directory did not contain the UDP Server and Client samples.
I did find the TCP Server and Client samples though.
Are the UDP samples in another directory or repo? Thank you.

**Comment by B03901108** [comment]
I found the provided TCP server and client similar to those in the following GitHub repo, based on the disassembly.
[STM32CubeH7/LwIP](https://github.com/STMicroelectronics/STM32CubeH7/tree/master/Projects/STM32H743I-EVAL/Applications/LwIP)

Does the above apply to the UDP server and client?

**Comment by shandianchengzi** [comment]
Yes, they are compiled from stm32 f429 LwIP examples, which can be imported when using STM32CubeMX. I'm very sorry that I missed it when uploading. You can now view it here: 
1. https://github.com/MCUSec/SEmu/tree/main/DataSet/fuzz_tests/f429/LwIP_UDP_Echo_Client
2. https://github.com/MCUSec/SEmu/tree/main/DataSet/fuzz_tests/f429/LwIP_UDP_Echo_Server

**Comment by B03901108** [comment]
Hi, @shandianchengzi. Thank you so much for the update!
